### PR TITLE
Feature/notebooks

### PR DESCRIPTION
**Pull request by EStorvik**
Created both a notebook for showing some of the basic image manipulation tools in DarIA, and the curvature correction procedure. I am not too happy with the curvature correction part and will create a class and remake the notebook soon. Additionally, some files related to the notebooks were improved upon.

### CODE REVIEW

**Comment by jwboth** [comment]
@EStorvik , what is your sense? Shall we make this PR a draft and you plan to work on this, or is this meant to be reviewed and merged before further updates will be made?

**Review by jwboth — CHANGES_REQUESTED** [review]
Nice! See minor comments below.

File: src/daria/utils/conversions.py [hunk 1]
```diff
@@ -16,6 +19,34 @@ def cv2ToSkimage(img: np.ndarray) -> np.ndarray:
...
+
+    Returns:
+        da.Image: converged image
+    """
+    img_gray = img.copy()
+    img_gray.img = cv2.cvtColor(img_gray.img, cv2.COLOR_BGR2GRAY)
+    return img_gray
+
+
+def BGR2RED(img: da.Image) -> da.Image:
```
[the remark this answers is not in the document]
Comment (EStorvik) [reply src/daria/utils/conversions.py:36]: Yes, this was my plan!

File: src/daria/utils/conversions.py [hunk 2]
```diff
@@ -16,6 +19,34 @@ def cv2ToSkimage(img: np.ndarray) -> np.ndarray:
     return img[:, :, ::-1]
```
[the remark this answers is not in the document]
Comment (EStorvik) [reply src/daria/utils/conversions.py:19]: Agreed

File: src/daria/image/image.py [hunk 3]
```diff
@@ -202,6 +203,15 @@ def show(self, name: Optional[str] = None, wait: Optional[int] = 0) -> None:
         cv2.waitKey(wait)
         cv2.destroyAllWindows()
 
+    def plt_show(self) -> None:
+        """Show image using matplotlib.pyplots built-in imshow"""
+
+        if len(self.img.shape) == 3:
```
[the remark this answers is not in the document]
Comment (EStorvik) [reply src/daria/image/image.py]: I think that this actually worked, but I will anyways remove it with the inclusions of the colorspace tag

**Comment by jwboth** [comment]
And remove the empty notebook.

**Review by jwboth — APPROVED** [review]
Whenever you think this is finished, feel free to merge!

**Pull request merged**